test/test_datasets.py

```diff
@@ -467,7 +467,7 @@ def inject_fake_data(self, tmpdir, config):
 class WIDERFaceTestCase(datasets_utils.ImageDatasetTestCase):
     DATASET_CLASS = datasets.WIDERFace
     FEATURE_TYPES = (PIL.Image.Image, (dict, type(None)))  # test split returns None as target
-    CONFIGS = datasets_utils.combinations_grid(split=('train', 'val', 'test'))
+    ADDITIONAL_CONFIGS = datasets_utils.combinations_grid(split=('val', 'test'))
```

Comment:
The implicit assumption here is  that the default is 'train'. If it ever changes (unlikely in this case but maybe not for other parameters?) then that value won't be tested anymore and we might not find out. Should we just leave these, as the duplicates will be removed anyway? no strong opinion

Reply:
I think you are right. We should not rely on the default values especially with duplicate removal.